src/core/sticky/StickyFooter.tsx

```diff
@@ -33,6 +37,10 @@ export default class StickyFooter<P extends StickyObjectProps> extends StickyObj
             this.bounceScrolling = this.hasReachedBoundary(offsetY, windowBound);
             if (largestVisibleIndex > stickyIndices[stickyIndices.length - 1] || this.bounceScrolling) {
                 this.stickyVisiblity = false;
+                //This is needed only in when the window is non-scrollable.
+                if (this.props.alwaysStickBottom && offsetY === 0) {
+                    this.stickyVisiblity = true;
+                }
```

Comment:
Again let's make this an outer check, and do the usual otherwise.

Reply:
Its not going to help as its a `and` condition and there no computation.

Reply:
Yea not computation wise but logical separation wise. But this is cool as well 👍